bundle-redhat/manifests/datadog-operator.clusterserviceversion.yaml

```diff
@@ -524,6 +538,21 @@ spec:
           - get
           - patch
           - update
+        - apiGroups:
+          - datadoghq.com
+          resources:
+          - datadogmetrics
+          verbs:
+          - create
+          - delete
+          - list
+          - watch
+        - apiGroups:
+          - datadoghq.com
+          resources:
+          - datadogmetrics/status
+          verbs:
+          - update
```

Comment:
IMO it is ok that on main we dont have what was released on the release branch. 
It can still help the testing before.
I can add change the version to `0.7.2-main` to be more clear

Reply:
Sounds good :+1:
Not sure if adding `0.7.2-main` brings any benefits. No strong opinion.